src/covid_model_seiir_pipeline/pipeline/diagnostics/task.py

```diff
@@ -73,6 +73,7 @@ def run_grid_plots(diagnostics_version: str, name: str, progress_bar: bool) -> N
             date_end=pd.to_datetime(grid_plot_spec.date_end),
             output_dir=plot_cache,
         )
+        _runner(locs_to_plot[0])
```

Comment:
wat?

Reply:
Validation leftovers.  Hard to debug multiprocessing code.